ranger/config/commands.py

```diff
@@ -1743,13 +1743,26 @@ class narrow(Command):
 
     Show only the files selected right now. If no files are selected,
     disable narrowing.
+
+    To disable files when selection is active, use the :unnarrow
+    command.
     """
+
     def execute(self):
-        if self.fm.thisdir.marked_items:
+
+        # by default, calling :narrow enables narrow mode
+        self.narrow_mode = True
+
+        # turn off narrow mode if called with ":narrow False"
+        if self.arg(1) and self.arg(1) == "False":
+            self.narrow_mode = False
```

Comment:
I think we can simplify this to `narrow_mode = self.arg(1).lower() != "false"`. Can be an if statement but I prefer not having to keep track of changing state.

Doesn't need to be an instance variable if it's only used in this method.

Reply:
The case lowering is because we tend to accept bools regardless of case.

Reply:
Great idea, I redid the patch. It makes the code much simpler.